environment.yml

```diff
@@ -3,15 +3,15 @@ channels:
   - conda-forge
 
 dependencies:
-  # Python
   - python >=3.6
-  # Required
   - pip
   - geopandas
   - matplotlib-base
   - psycopg2
-  - postgis
```

Comment:
Hrmf. I would really like to be able to do something like

```yaml
 - postgis # [not win]
```

to allow installation for non-Windows platforms which actually have the package available. But on closer inspection, support for this in `environment.yml` files seems to be something for a work in progress, so I guess we need to take the dependency out completely.

However, it would probably be helpful to mention in the geoDB installation documentation that PostgreSQL and PostGIS can be installed with conda/mamba on non-Windows platforms.

Reply:
Yes, but: the `postgis` package is not needed actually. I think that `postgis` might have been added as dependency because it ensured that the (required) dependency `psycopg2` is availabe transitively. So there is no need to discriminate between Windows and non-Windows.